### PR TITLE
Sample scrolldepth events

### PR DESCRIPTION
**Pull request by tom-parker**
cc @andygout @ironsidevsquincy @commuterjoy 

I'm hoping this is ok to sample the scroll depth events from the frontpage. We saw ~400K of these from the front page yesterday which is almost a third of our daily allowance.

### CODE REVIEW

**Comment by andygout** [comment]
What proportion of the existing scroll depth events will we continue to catch - 10% (I think I remember being mentioned)?  I'd imagine that 40K per day would still be sufficient data.

**Comment by tom-parker** [comment]
Think it works out ~6.5%

File: client/components/scroll-depth/scroll-depth.js [hunk 1]
```diff
@@ -23,6 +23,9 @@ const scrollHandlerFactory = () => {
 
 export default () => {
 	const scrollHandler = scrollHandlerFactory();
+	if (!/spoor-id=0/.test(document.cookie)) {
```
Comment (ironsidevsquincy) [inline client/components/scroll-depth/scroll-depth.js]: any reason not to put this above the above line?

Comment (tom-parker) [inline client/components/scroll-depth/scroll-depth.js]: Nope!

**Comment by ironsidevsquincy** [comment]
:+1:

**Pull request merged**